.github/workflows/terraform.yml

```
@@ -0,0 +1,39 @@
+name: 'Terraform'
+
+on:
+  pull_request:
+    branches:
+      - master
+
+env:
+  TF_VERSION: 0.12.29
```

use a newer version for testing, e.g. 
```suggestion
  TF_VERSION: 0.14.3
```